### ISSUE TITLE
webpack 5 has outputed incompatible `export.default`

### ISSUE THREAD

**Issue by aleen42** [webpack/webpack#13971]
# Bug report

**What is the current behavior?**

As we all known, `exports.default` is not compatible with ES3 syntax, and developers always use `babel` to convert it into `exports['default']`, but recently I found that Webpack 5 has incorrectly converted `exports['default']` again into `exports.default`, and is there any workaround?

**If the current behaviour is a bug, please provide the steps to reproduce.**

To put it simply, we can compile a module with Webpack directly without Babel:

```js
exports['default'] = {}
```

and you can find that after Webpack compilation, it has been converted to:

```js
exports.default = {};
```

**What is the expected behaviour?**

Do not convert it.

**Other relevant information:**
webpack version: 5.48.0
Node.js version: 13.14.0
Operating System: Windows

**Comment by aleen42** [comment]
Followed the code and found that whether preserved words is safe under a ES3 syntax environment?

https://github.com/webpack/webpack/blob/400a0f94ab45ca20b10f219c8311e87d3d3f108c/lib/util/propertyAccess.js#L8

**Comment by alexander-akait** [comment]
We don't support es3

**Comment by aleen42** [comment]
@alexander-akait It is not the problem about whether Webpack support ES3, it is the problem that Webpack has made any libraries supported under ES3 broken after compilation.

**Comment by aleen42** [comment]
@alexander-akait what is the problem if outputting es3 styles? unnecessary code to maintain? The best solution is not to change the original source.

**Comment by alexander-akait** [comment]
> @alexander-akait what is the problem if outputting es3 styles? unnecessary code to maintain?

Yes, we don't want to support outdated ES3

**Comment by alexander-akait** [comment]
We can merge your PR, but we do not want to guarantee ES3 compatibility

**Comment by aleen42** [comment]
@alexander-akait  I know your thought, and as mentioned above, it is smart to implement a complicated detection to solve the problem, but it should be a height overhead.

**Comment by alexander-akait** [comment]
Fixed https://github.com/webpack/webpack/pull/13972/
